src/test/java/seedu/address/storage/JsonAdaptedTaskTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.TypicalTasks.CALL_EMBASSY;
+import static seedu.address.testutil.TypicalTasks.REMIND_PATIENT;
+import static seedu.address.testutil.TypicalTasks.RESTOCK_SUPPLY;
+
+import org.junit.jupiter.api.Test;
+
+class JsonAdaptedTaskTest {
+    @Test
+    public void toModelType_returnsTask_remindPatient() throws Exception {
```

Comment:
you might want to standardize the naming of methods, so maybe this method can be `toModelType_sameModelType_success()`. same for subsequent methods below `toModelType_sameModelType_throwsException()`, `toModelType_sameModelType_failure()`